src/main/groovy/edu/oregonstate/mist/catalogapitest/mapper/InstructorMapper.groovy

```diff
@@ -0,0 +1,25 @@
+package edu.oregonstate.mist.catalogapitest.mapper
+
+import edu.oregonstate.mist.catalogapitest.core.Instructor
+import org.skife.jdbi.v2.StatementContext
+import org.skife.jdbi.v2.tweak.ResultSetMapper
+import java.sql.ResultSet
+import java.sql.SQLException
+
+/**
+ * Mapper for Instructor object properties and database columns.
+ */
+public class InstructorMapper implements ResultSetMapper<Instructor> {
+    public Instructor map(int i, ResultSet rs, StatementContext sc) throws SQLException {
+        Instructor instructor = new Instructor()
+
+        instructor.with {
+            cid               = rs.getInt     'CID'
+            firstInitial      = rs.getString  'FIRST_INITIAL'
+            lastName          = rs.getString  'LAST_NAME'
+            numberOfCourses   = rs.getInt     'NUMBER_OF_COURSES'
+        }
+
+        return instructor
```

Comment:
groovy's [named argument constructor](http://docs.groovy-lang.org/latest/html/documentation/index.html#_named_argument_constructor) enables you to initialize the properties `foo` and `bar` of `SomeClass` like

    SomeClass var = new SomeClass(foo: 'a value', bar: 'another value')

groovy's `.with()` syntax is equivalent to calling a sequence of setters like

    var.setFoo('a value')
    var.setBar('another value')

my suggestion was to instantiate, initialize, and return all in one line like

    return new SomeClass(foo: 'a value', bar: 'another value')